render/tests/test-updateNodesFuzzer.js

```diff
@@ -25,18 +25,44 @@ o.spec("updateNodes keyed list Fuzzer", function() {
 		var tests = 250
 
 		while (tests--) {
-			var test = fuzzTest(c.delMax, c.movMax, c.insMax)
-			o(i++ + ": " + test.list.join() + " -> " + test.updated.join(), function() {
+			const test = fuzzTest(c.delMax, c.movMax, c.insMax)
+			const id = i++
+			o(id + ": " + test.list.join() + " -> " + test.updated.join(), function() {
 				render(root, test.list.map(function(x){return m(x, {key: x})}))
 				addSpies(root)
 				render(root, test.updated.map(function(x){return m(x, {key: x})}))
 
-				if (root.appendChild.callCount + root.insertBefore.callCount !== test.expected.creations + test.expected.moves) console.log(test, {aC: root.appendChild.callCount, iB: root.insertBefore.callCount}, [].map.call(root.childNodes, function(n){return n.nodeName.toLowerCase()}))
+				// FIXME: This does not take into account the "swaps and list reversals" heuristic in updateNodes().
+				// if (root.appendChild.callCount + root.insertBefore.callCount !== test.expected.creations + test.expected.moves) console.log(test, {aC: root.appendChild.callCount, iB: root.insertBefore.callCount}, [].map.call(root.childNodes, function(n){return n.nodeName.toLowerCase()}))
```

Comment:
Could you elaborate on what the specific issue is with this?

This feels a bit too generic. Given the nature of the issue it's checking, and the fact it's just auditing for perf opportunities, it might take years to get fixed, assuming it gets fixed at all.

Reply:
You're absolutely right. I'd like to leave a more detailed comment.